modules/nf-core/lofreq/callparallel/tests/main.nf.test

```diff
@@ -34,7 +34,7 @@ nextflow_process {
                 { assert process.success },
                 { assert snapshot(
                     process.out.versions,
-                    process.out.tbi.collect { file(it[1]).getName() }
+                    process.out.tbi
```

Comment:
```suggestion
                    process.out.tbi,
                    path(process.out.vcf[0][1]).vcf.summary,
					path(process.out.vcf[0][1]).vcf.variantsMD5
```

Reply:
Oh I didn't really know this could be done this way.